bin/w2a

```diff
@@ -50,9 +51,11 @@ if ReverseAsciidoctor.config.external_images && ReverseAsciidoctor.config.destin
 end
 
 ReverseAsciidoctor.config.sourcedir = Dir.mktmpdir
+# TODO: Check if needed
```

Comment:
Placing `TODO` is good practice, alongside with this I propose to report an issue also

Because sometimes those todo being ignored

Reply:
#40